config/crd/bases/toolchain.dev.openshift.com_tiertemplates.yaml

```
@@ -76,6 +75,8 @@ spec:
                       can be included in the output.
                     type: string
                   metadata:
+                    description: 'metadata is the standard object''s metadata. More
```

Extra comma? Not sure whether it'd effect anything but just thought I'd raise it

```suggestion
                    description: 'metadata is the standard object's metadata. More
```

---

It's a generated yaml. Not sure what API go code exactly is the source of this string but we shouldn't modify it in the CRD yaml directly :)